src/views/Workflow.vue

```diff
@@ -227,10 +304,17 @@ export default {
         if (this.deltaSubscriptions.length === 0) {
           this.$workflowService.stopDeltasSubscription()
           this.tree.clear()
+        } else if (vm.deltaTableSubscriptions.includes(subscriptionId)) {
+          // if this is a table widget with a deltas subscription, then stop it if the last widget using it
+          vm.deltaSubscriptions.splice(this.deltaSubscriptions.indexOf(subscriptionId), 1)
+          if (this.deltaTableSubscriptions.length === 0) {
+            this.$workflowService.stopTableDeltasSubscription()
```

Comment:
`.stopTableDeltasSubscription` does not exist yet. So when we we removed the table widget, the subscription never stops.

You can confirm that by looking at the WebSockets messages that never cease even when you don't have any widgets (note: subscription `id: 1` is the GScan component... in this case, `2` was the default tree, and now `3` contains the table widget that I added, and then removed... `2` received a stop and is over, while `3` never stopped).